build.sbt

```diff
@@ -159,7 +160,9 @@ lazy val testing = (crossProject(JSPlatform, JVMPlatform) in file ("testing"))
     prevVersions := {
       scalaBinaryVersion.value match {
         case "2.10" | "2.11" | "2.12" =>
-          Set("1.5.0", "1.6.0", "1.6.1", "1.7.0", "1.8.0", "1.8.1")
+          Set("1.5.0", "1.6.0", "1.6.1", "1.7.0", "1.8.0", "1.8.2")
+        case "2.13" =>
+          Set("1.8.2")
```

Comment:
There's an opportunity to pull out common revisions here similar to core, but I took the easy way out for now.